truffle-config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  networks: {
+    ganache: {
+      host: "127.0.0.1",
+      port: 7545,
+      network_id: "*",
+    },
+  },
```

Comment:
We need to add a compiler object here for the specific Solidity version. That will get rid of the warnings.

Reply:
Awesome, thank you!